plugin/vim-gh-line.vim

```diff
@@ -27,7 +27,7 @@ endif
 if !exists('g:gh_open_command')
     if has('win16') || has('win32') || has('win64')
         let g:gh_open_command = 'start '
-    elseif has('mac') || has('macunix') || has('gui_macvim')
+    elseif has('mac') || has('macunix') || has('gui_macvim') || executable('open')
```

Comment:
`open` command might be present on Linux, and it won't work: https://unix.stackexchange.com/a/253432

> otherwise doesn't work in vim in macOS

Perhaps there's an issue with your environment? Works fine for me. `:echo has('mac')` should print `1` on macos.

Reply:
what version of vim did you test and what version of macOS do you have?

Reply:
hmm, after recent update I have `has('mac')` as `1` also. weird